doc/classes/LinkButton.xml

```diff
@@ -15,8 +15,10 @@
 		<member name="focus_mode" type="int" setter="set_focus_mode" getter="get_focus_mode" override="true" enum="Control.FocusMode" default="0" />
 		<member name="mouse_default_cursor_shape" type="int" setter="set_default_cursor_shape" getter="get_default_cursor_shape" override="true" enum="Control.CursorShape" default="2" />
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
+			The button's text that will be displayed inside the button's area.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
+			The underline mode, determines when to show the underline. See [enum UnderlineMode] constants.
```

Comment:
Change to -> "Determines when to show the underline. See [enum UnderlineMode] for options."